api/src/main/scala/io/phdata/startup/ADGroupsSynchronizer.scala

```diff
@@ -39,7 +39,7 @@ class ADGroupsSynchronizer[F[_]: Sync: Timer: ContextShift](context: AppContext[
       registrations <- ldapRegistrations
       _ <- registrations.traverse { lDAPRegistration =>
         for {
-          lDAPUsers <- context.provisioningLDAPClient.groupMembers(lDAPRegistration.distinguishedName)
+          lDAPUsers <- context.lookupLDAPClient.groupMembers(lDAPRegistration.distinguishedName)
```

Comment:
This looks like you fixed a bug here (even though it's a behavior change). I think this was not running correctly before on certain sites